fiftyone/core/config.py

```diff
@@ -375,6 +396,25 @@ def get_colormap(self, colorscale=None, n=256, hex_strs=False):
         return fop.get_colormap(colorscale, n=n, hex_strs=hex_strs)
 
     def _init(self):
+        color_by_value = self.parse_bool(
+            {},
+            "color_by_value",
+            env_var="FIFTYONE_APP_COLOR_BY_VALUE",
+            default=False,
```

Comment:
I find it unlikely someone is using this, so I personally don't care if this is just silently ignored.

If you do leave this though, setting `default=None` would be a bit better so you can warn if they've hard-coded `False` in their config.